### PR TITLE
Permissions docs changes

### PR DESCRIPTION
**Pull request by samnixon87**
Tweak permissions docs structure to make it cleaner. Move Backstage referenced docs out into a separate section

https://app.shortcut.com/larder/story/23840/policy-docs-snags

### CODE REVIEW

File: content/docs/details/permissions/index.md [hunk 1]
```diff
@@ -6,46 +6,31 @@ description: How to configure access control permissions in Roadie.
 
 ## Introduction
 
-Roadie comes with RBAC out of the box. We provide 4 default roles: `admin`, `viewer`, `maintainer` and `tech-insights-admin`. It is possible to also define custom roles - please contact our sales team if this is something you are interested in.
+Roadie permissions allow rules to be set about which users have acces to which features and information.
```
Comment (punkle) [inline content/docs/details/permissions/index.md:9]: We are referring to this feature as rbac on the managing-roles page.

Comment (punkle) [reply content/docs/details/permissions/index.md:9]: also type on acces

File: content/docs/details/permissions/index.md [hunk 2]
```diff
@@ -6,46 +6,31 @@ description: How to configure access control permissions in Roadie.
...
-| admin               | Can perform any action in Roadie                                                                                               |
-| viewer              | Can only view data within Roadie                                                                                               |
-| maintainer          | Can read and write data, but cannot access administration settings or tech insights editing                                    |
-| tech-insights-admin | Can edit scorecards, checks and data sources in tech insights. This role requires one of viewer / maintainer to work correctly |
+This can be set at a coarse level (i.e. whole features of the application) or at a fine-grain level (i.e. the ability to view specific entities within the catalog).
 
-## Assigning Roles
+A good example of a permission is the ability to allow users to see a Scaffolder Template. Let's say this Template has the ability to change a cloud budget for a given service. Not everyone within an organisation may have that ability. A permissions attached to that Template would allow the team governing Roadie to limit access to that Template.
```
Comment (punkle) [inline content/docs/details/permissions/index.md:13]: maybe lower case template or make it italic?

File: content/docs/details/permissions/index.md [hunk 3]
```diff
@@ -6,46 +6,31 @@ description: How to configure access control permissions in Roadie.
...
-If your Roadie tenant is using a custom identity provider, you can choose to include the roles field in your id token as follows:
+## Backstage Permissions Framework
 
-```json
-{
-  "sub": "77d0c4cb-706c-4aa4-b18e-bed538a33aa7",
-  "roles": ["viewer", "tech-insights-admin"]
-}
-```
+Roadies Permissions system is based on the Bacsktage Permissions framework. More information on the framework in general can be found in Backstage docs:
```
Comment (punkle) [inline content/docs/details/permissions/index.md:33]: Again we are referring to the feature in different ways in different places.

File: content/docs/details/permissions/index.md [hunk 4]
```diff
@@ -6,46 +6,31 @@ description: How to configure access control permissions in Roadie.
...
-
-If a user has no role associated yet, e.g. if it is the first time they have logged in and there are no roles associated with the user, then the user is allowed the policies associated with the default roles. There can be many default roles.
-
-To edit the default roles:
-
-- Visit `http://<tenant name>.roadie.so/administration/settings/roles`
-- Click the edit pencil beside the roles you would like to be the default roles
-- The click "Set as default role" and save.
-- You can perform similar steps to make a role no longer a default role.
+- Backstage docs on [policy definitions](https://backstage.io/docs/permissions/writing-a-policy/).
```
Comment (punkle) [inline content/docs/details/permissions/index.md:35]: I believe these links are irrelevant, we have implemented our own permissions policy.. I would suggest removing these links. I think these are docs that integrators would use.. and we wouldnt expect our users to have this context. IMO thoughts?

Comment (samnixon87) [reply content/docs/details/permissions/index.md:35]: What I want to indicate is (mainly for self-hosted folks with custom plugins) that the permissions framework is there and our work is based on it. We deviate + you don't need to understand permissions in order to understand our stuff, but it's useful context

Comment (punkle) [reply content/docs/details/permissions/index.md:35]: coola boola

**Review by punkle — APPROVED** [review]

**Pull request merged**